File: p4/4.1.py
from funk import *
max = 5

def main():
    re = True
    while re == True:
        size = inputh("Ile elemntuw chcesz podac")
        tab = [0,]*size
        czytajt(tab,size)
        piszt(tab,size)
        print(f"Twoje liczby odwrotnie to :",end="")
        for i in reversed(tab):
            print(f"{i},", end="")
        print()
        re = jeszcze()

if __name__ == "__main__":
    main()

File: s4/4.1.cpp
#include "funk.h"
#define max 100
using namespace std;

int main(){
    srand(time(NULL));
    bool re = true;
    do{
        int size = input("Ile elementow chcesz podac:");        
        int tab[max];
        czytajt(tab, size);
        piszt(tab, size);
        cout<<"Twoje liczby odwrotnie to: ";
        for(int i=0;i<size;i++){
            cout<<tab[size-i-1]<<" ";
        }
        jeszcze(re);
    }while(re==true);
    return 0;
}
